app/models/evidence_item.rb

```diff
@@ -74,6 +74,10 @@ def self.advanced_search_scope
     eager_load(:submitter, :disease, :source, :drugs, :open_changes, variant: [:gene, :variant_aliases])
   end
 
+  def self.variant_group_scope
+    eager_load(variant: [:variant_groups])
```

Comment:
You'll likely want to include the `index_scope` here as well as the joins you need for the variant group query. In theory, you'll know you've gotten it correct if you look at the server logs and see only one query (or a handful) rather than 1 per evidence item. 

Making `variant_group_scope` something like ` eager_load(:disease, :source, :drugs, :open_changes, variant: [:variant_groups])` would probably be a good place to start!